mentorApp/.vscode/launch.json

```diff
@@ -2,30 +2,32 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
```

Comment:
Not sure how important it is to keep the `launch.json` file. We would need all developers to change the `deviceIds` of their android and iphone emulators to use this file, and VSC flutter extension handles all this for me.

Reply:
Agreed! I don't think we need this file at all. We can get rid of it next time :)